InternetHospitalWebUI/src/app/Models/Filter.ts

```diff
@@ -0,0 +1,14 @@
+export class Filter {
+    selectedSpecialization: string = "";
+    searchKey: string;
+    isWithParams: boolean;
+  
+    public CheckIfPropertyExist():void {
+      if (this.searchKey || this.selectedSpecialization) {
+        this.isWithParams = true;
+      }
+      else {
+        this.isWithParams = false;
+      }
+    }
```

Comment:
Is it possible to better use the ternary operator here? There will be only one line

Reply:
Moreover just this.isWithParams = this.searchKey || this.SelectedSpecialization;

Reply:
'this.isWithParams = this.searchKey || this.selectedSpecialization', this operation trying to set string value into this.isWithParams

Reply:
Changed signature like Serhii said